crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -66,10 +66,12 @@ def restore_chef
       def restore_crowbar
         Crowbar::Backup::Base.restore_files.each do |source, destination|
           # keep the permissions of the files that are already in place
+          srcpath = @data.join("crowbar", source)
```

Comment:
and delete the comment above. this will hopefully change with https://github.com/crowbar/crowbar-core/pull/199